src/api/api.data.ts

```diff
@@ -57,22 +60,39 @@ function parsePageHeaders(link: string) {
   const h2s = src.match(/^## [^\n]+/gm)
   let headers: APIHeader[] = []
   if (h2s) {
+    const anchorRE = /\{#([^}]+)\}/
```

Comment:
I changed this RegExp. In practice, the previous RegExp worked fine for the current headers in the API, but it assumed that the anchors can only contain letters, numbers and hyphens. We already have headers elsewhere in the docs that use other characters and future additions to the API may do the same.